core/src/query.rs

```diff
@@ -85,8 +83,10 @@ impl MongoQuery {
             doc! {"$group": {
                 "_id": null,
                 "collections": {
-                    "collectionName": "$_id",
-                    "schema": "$schema"
+                    "$push": {
+                        "collectionName": "$_id",
+                        "schema": "$schema"
+                    }
```

Comment:
In stages like $group, an accumulator must be used ($push does the job here). This was throwing an error from MongoDB.